woodwork/tests/accessor/test_table_accessor.py

```diff
@@ -1019,6 +1068,18 @@ def test_get_subset_df_with_schema(sample_df):
     validate_subset_schema(transfer_schema.ww.schema, schema)
 
 
+def test_use_dataframe_order(sample_df):
```

Comment:
Maybe update this test name to `test_get_subset_df_use_dataframe_order` to make it more clear what this test is about.

Reply:
✅  8126f37